conf/server.yaml.template

```diff
@@ -3,6 +3,13 @@ SERVER:
   HOSTNAMES: []
   #  - replace.with.satellite.hostname
   #  - replace.with.satellite.hostname
+  # default os name and versions used in default_os fixture
+  DEFAULT_OS:
```

Comment:
Lets not put this in `server.yaml` as it is not related to Satellite server

Reply:
`robottelo.yaml` is definitely better place.

Others are suggesting `supportability.yaml` and you can see this relates more to `ContentHosts`
Or even brand new `os.yaml` would work for me.

BTW I've found orphaned `distros.yaml`  :smile:

Reply:
I had it in `robottelo.yaml` first but could not find any validators for that.

I did not want to create a new settings file (to much effort)

Would everybody be happy if I put it into [`distros.yaml`](https://github.com/SatelliteQE/robottelo/blob/master/conf/distro.yaml.template) or [supportability.yaml](https://github.com/SatelliteQE/robottelo/blob/master/conf/supportability.yaml)?

Reply:
@dosas Why would you need to even add this setting since supportability.yaml is giving you all the matrix that you need to test ?